static/src/js/components/TemporalDisplay/__tests__/TemporalSelectionDropdown.test.js

```diff
@@ -1,147 +1,210 @@
   })
 
-  test('clears the values onClearClick', () => {
-    const { enzymeWrapper } = setup()
+  test('clears the values onClearClick', async () => {
     const onChangeQueryMock = jest.fn()
-    enzymeWrapper.setProps({
+    const user = userEvent.setup()
+
+    const validEndDate = '2019-03-30T00:00:00.000Z'
+    const validStartDate = '2019-03-29T00:00:00.000Z'
+
+    const { getByRole, getAllByRole } = setup({
       onChangeQuery: onChangeQueryMock,
       temporalSearch: {
-        endDate: '2019-03-30T00:00:00.000Z',
-        startDate: '2019-03-29T00:00:00.000Z'
+        endDate: validEndDate,
+        startDate: validStartDate
       }
     })
-    enzymeWrapper.setState({ open: true })
 
-    enzymeWrapper.instance().onClearClick()
+    await waitFor(async () => {
+      await user.click(getByRole('button'))
+    })
 
-    expect(onChangeQueryMock).toHaveBeenCalledWith({
-      collection: {
-        temporal: {}
-      }
+    const inputs = getAllByRole('textbox')
+    expect(inputs[0].value).toBe(moment.utc(validStartDate).format('YYYY-MM-DD HH:mm:ss'))
+    expect(inputs[1].value).toBe(moment.utc(validEndDate).format('YYYY-MM-DD HH:mm:ss'))
+
+    const clearBtn = getAllByRole('button', { name: 'Clear' })[2]
+    await waitFor(async () => {
+      await user.click(clearBtn)
     })
-    expect(onChangeQueryMock).toHaveBeenCalledTimes(1)
-    expect(enzymeWrapper.state()).toEqual({
-      disabled: false,
-      open: false,
-      temporal: {
-        startDate: '',
-        endDate: '',
-        recurringDayEnd: '',
-        recurringDayStart: '',
-        isRecurring: false
-      }
+
+    await waitFor(async () => {
+      await user.click(getByRole('button'))
     })
+
+    const updatedInputs = getAllByRole('textbox')
+    expect(updatedInputs[0].value).toBe('')
+    expect(updatedInputs[1].value).toBe('')
+
+    expect(onChangeQueryMock).toHaveBeenCalledTimes(1)
   })
 
-  test('applies the values onApplyClick', () => {
-    const { enzymeWrapper } = setup()
+  test('applies the values onApplyClick', async () => {
     const onChangeQueryMock = jest.fn()
-    enzymeWrapper.setProps({
-      onChangeQuery: onChangeQueryMock,
-      temporalSearch: {}
-    })
-    enzymeWrapper.setState({
-      open: true,
-      temporal: {
-        endDate: '2019-03-30T00:00:00.000Z',
-        startDate: '2019-03-29T00:00:00.000Z'
-      }
+    const user = userEvent.setup()
+
+    setup({
+      onChangeQuery: onChangeQueryMock
     })
 
-    enzymeWrapper.instance().onApplyClick()
+    const validEndDate = '2019-03-30T00:00:00.000Z'
+    const validStartDate = '2019-03-29T00:00:00.000Z'
+
+    fireEvent.click(screen.getByRole('button'))
+
+    const inputs = screen.getAllByRole('textbox')
+
+    fireEvent.change(inputs[0], { target: { value: validStartDate } })
+    fireEvent.change(inputs[1], { target: { value: validEndDate } })
+
+    expect(inputs[0].value).toBe(moment.utc(validStartDate).format('YYYY-MM-DD HH:mm:ss'))
```

Comment:
this is the other spot where I would think you would not need to call moment and convert time
```suggestion
    expect(inputs[0].value).toBe(validStartDate)
```

Reply:
and the one below on line 190